### ISSUE TITLE
After following instructions on the github page, dependencies are still unmet

### ISSUE THREAD

**Issue by pixelpax** [flyser/cournal#21]
I'm getting the following traceback related to the twisted package installed by the script:

```
Traceback (most recent call last):
  File "./cournal.py", line 22, in <module>
    from cournal.run import run
  File "/media/gabriel/Windows8_OS/Users/Gabriel/Downloads/cournal/cournal/run.py", line 28, in <module>
    from cournal.mainwindow import MainWindow
  File "/media/gabriel/Windows8_OS/Users/Gabriel/Downloads/cournal/cournal/mainwindow.py", line 29, in <module>
    from cournal.document.document import Document
  File "/media/gabriel/Windows8_OS/Users/Gabriel/Downloads/cournal/cournal/document/document.py", line 26, in <module>
    from cournal.document.page import Page
  File "/media/gabriel/Windows8_OS/Users/Gabriel/Downloads/cournal/cournal/document/page.py", line 22, in <module>
    from cournal.document.stroke import Stroke
  File "/media/gabriel/Windows8_OS/Users/Gabriel/Downloads/cournal/cournal/document/stroke.py", line 22, in <module>
    from twisted.spread import pb
ImportError: No module named 'twisted.spread'
```

It should also be noted that the installation instructions on the project site are different from those on the github page: those on the github page include instructions to run the download-twisted.py script and some extra dependencies. 

Looking forward to trying cournal once this is sorted out :D

**Comment by flyser** [comment]
Could you delete the /media/gabriel/Windows8_OS/Users/Gabriel/Downloads/cournal/cournal/t3k folder, rerun the download-twisted script (did you run it in the first place?) and post its output?

**Comment by flyser** [comment]
wait ... does the filesystem support symlinks? Looks like you are using ntfs or fat ...

**Comment by HeroicAutodidact** [comment]
Hey Fabian,

Thanks for the response! My drive is ntfs formatted, but I have working
symlinks on it.

The output that I get after removing t3k and re-running the script:

```
/home/gabriel/Downloads/cournal
destination directory: t3k
requesting all changes
adding changesets
adding manifests
adding file changes
added 16429 changesets with 39024 changes to 5298 files
updating to branch t3k
1459 files updated, 0 files merged, 0 files removed, 0 files unresolved
Successfully downloaded twisted for python3
```

Please let me know if there's any other information I can supply which
would be useful to you.

**Comment by HeroicAutodidact** [comment]
Also, I did run the script before messaging you, but I would note that the
instructions on the website do not list it as a step, so I believe I used
pip3 to try to resolve dependencies before I noticed the discrepancy.

**Comment by flyser** [comment]
Could you try to reproduce it on a real linux filesystem? anything but fat32 or ntfs

**Comment by HeroicAutodidact** [comment]
OK, I just tried to reproduce it in my home directory which is ext4.

When I do this and attempt to run the download twisted script I get the
following:

```
/home/gabriel/cournal
destination directory: t3k
requesting all changes
adding changesets
adding manifests
adding file changes
...
```
At which point it hangs indefinitely (as far as I can tell) and I have to
interrupt it.

**Comment by HeroicAutodidact** [comment]
I've tried this on two linux machines, installing twisted using the script. I've also tried installing twisted using apt-get and pip3 to no avail (`ImportError: No module named 'twisted.spread'`). Perhaps it's something I'm doing wrong or an issue with twisted. Was there a change in their module structure lately? Python isn't my best language, I'm having a bit of trouble trying to understand why the spread module isn't appearing.
